rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
It won't work without it for now if we remove it. I plan to remove that in a later commit.